packages/server/src/orchestrator.ts

```diff
@@ -61,10 +54,10 @@ export function* createOrchestrator(options: OrchestratorOptions): Operation {
   })
 
   yield fork(createProxyServer({
+    agentServerConfig,
     delegate: proxyServerDelegate,
     port: options.project.proxy.port,
     targetPort: options.project.app.port,
-    inject: `<script src="${agentServerConfig.harnessUrl()}"></script>`,
```

Comment:
I decided to remove this argument. Previously the proxy server was just concerned with injecting something into the proxied page, but now it is actually much more specific to bigtest anyway, so we might as well generate this in the proxy server itself.